apps/ledger-live-desktop/src/renderer/families/celo/UnlockFlowModal/steps/StepAmount.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+import invariant from "invariant";
+import React from "react";
+import { Trans } from "react-i18next";
+import { SyncSkipUnderPriority } from "@ledgerhq/live-common/lib/bridge/react";
+
+import { urls } from "~/config/urls";
+
+import AccountFooter from "~/renderer/modals/Send/AccountFooter";
+import TrackPage from "~/renderer/analytics/TrackPage";
+import Box from "~/renderer/components/Box";
+import Button from "~/renderer/components/Button";
+import Alert from "~/renderer/components/Alert";
+import ErrorBanner from "~/renderer/components/ErrorBanner";
+
+import type { StepProps } from "../types";
+import AmountField from "../fields/AmountField";
+
+export default function StepAmount({
+  account,
+  parentAccount,
+  onChangeTransaction,
+  transaction,
+  status,
+  error,
+  bridgePending,
+  t,
+}: StepProps) {
+  invariant(account && transaction, "account and transaction required");
+
+  return (
+    <Box flow={1}>
+      <SyncSkipUnderPriority priority={100} />
+      <TrackPage category="Unlock Flow" name="Step 1" />
+      {error && <ErrorBanner error={error} />}
+      <Alert
+        type="primary"
+        learnMoreUrl={urls.stakingCelo}
+        learnMoreLabel={<Trans i18nKey="celo.unlock.steps.amount.learnMore" />}
+        mb={4}
+      >
+        Unlocked assets can be withdrawn after the 3-day unlocking period.
```

Comment:
Should this text be added to the i18n files? ☝️

Reply:
thanks! 👍